bmeg_app/__main__.py

```diff
@@ -0,0 +1,171 @@
+import bmeg_app.appLayout as ly
+from bmeg_app.views import app_about, app_tcga, app_lit, app_repurpose_drug
+import base64
+import dash
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+
+# Layout 
+styles=ly.styles
+
+# Images
+image_filename = 'bmeg_app/images/bmeg_logo.png' 
+encoded_image3 = base64.b64encode(open(image_filename, 'rb').read())
+ 
+ 
+# we use the Row and Col components to construct the sidebar header
+# it consists of a title, and a toggle, the latter is hidden on large screens
+sidebar_header = dbc.Row(
+    [
+        dbc.Col(
+            html.Div(html.Img(src='data:image/png;base64,{}'.format(encoded_image3.decode()),
+                style={'height':'85%','width':'85%', 'marginTop': 0, 'marginBottom':0}),
+            ),
+            className="display-4"),
+        dbc.Col(
+            [
+                html.Button(
+                    # use the Bootstrap navbar-toggler classes to style
+                    html.Span(className="navbar-toggler-icon"),
+                    className="navbar-toggler",
+                    # the navbar-toggler classes don't set color
+                    style={
+                        "color": "rgba(0,0,0,.5)",
+                        "border-color": "rgba(0,0,0,.1)",
+                    },
+                    id="navbar-toggle",
+                ),
+                html.Button(
+                    # use the Bootstrap navbar-toggler classes to style
+                    html.Span(className="navbar-toggler-icon"),
+                    className="navbar-toggler",
+                    # the navbar-toggler classes don't set color
+                    style={
+                        "color": "rgba(0,0,0,.5)",
+                        "border-color": "rgba(0,0,0,.1)",
+                    },
+                    id="sidebar-toggle",
+                ),
+            ],
+            # the column containing the toggle will be only as wide as the
+            # toggle, resulting in the toggle being right aligned
+            width="auto",
+            # vertically align the toggle in the center
+            align="center",
+        ),
+    ]
+)
+
+sidebar = html.Div(
+    [
+        sidebar_header,
+        # we wrap the horizontal rule and short blurb in a div that can be
+        # hidden on a small screen
+        html.Div(
+            [
+                html.Hr(),
+                html.P(
+                    "A graph database for "
+                    "merging and analyzing connected data",
+                    className="lead",
+                    style={'font-size':styles['textStyles']['size_font_card'],'fontFamily':styles['textStyles']['type_font']},
+                ),
+            ],
+            id="blurb",
+        ),
+        # use the Collapse component to animate hiding / revealing links
+        dbc.Collapse(
+            dbc.Nav(
```

Comment:
We should figure out how to get these kinds of title stored into a config file

Reply:
Maybe using a library like https://pypi.org/project/python-i18n/
@bwalsh any thoughts about a good text config library to use?